float64 velocity
float64 damping_distance
Waypoint[] mission_waypoint
